docs/network.md

```diff
@@ -206,6 +206,8 @@ The following proposal types are currently supported:
 
 - `upgrade <descriptor.json>`: Network upgrade proposal. Provide a JSON file
   containing the upgrade descriptor.
+- `parameter-change <module-name> <changes.json>`: Network parameter change
```

Comment:
Let's order the proposal names by name (cancel-upgrade, parameter-change, upgrade).